### DISCUSSION TITLE
FPS lost

### DISCUSSION THREAD

**Discussion by RoseLuck** [Hekili/hekili#1031] [Q&A] [answered]
Good afternoon everyone.
When using Нekili, FPS drops sharply from 60 to 40-45.

**Comment by Hekili** [comment]
The addon will use CPU. If it uses too much for you/your spec, then don't use it.

**Reply by RoseLuck** [reply]
Everything was fine before. It all started just a week ago.
ryzen 7 1700 / gtx 1060 6gb

**Reply by Hekili** [reply]
If you feel that something has changed and it has impacted performance for you, that belongs in an issue ticket and not the discussion board.

**Reply by Rafa28062013** [reply]
> Everything was fine before. It all started just a week ago.
> ryzen 7 1700 / gtx 1060 6gb

I have that Problem too

**Reply by Hekili** [reply]
Same answer.

**Comment by Hekili** [comment, marked as answer]
The performance impact of the addon should've improved since this thread began.  I found that the addon had been updating recommendations more frequently than necessary, and reduced the default threshold (unless something has happened that the addon should care about).  You may or may not notice an improvement.

Some specs (i.e., Fire Mage) will still use more CPU than the others.
